app/src/main/java/org/schabi/newpipe/fragments/local/BaseLocalListFragment.java

```diff
@@ -86,19 +86,30 @@ protected void initListeners() {
     // Lifecycle - Menu
     //////////////////////////////////////////////////////////////////////////*/
 
+    /** Determines if the fragment is part of the main fragment view pager.
+     * If so, then this method must be overriden to return true
+     * in order to show the hamburger menu. */
+    protected boolean isPartOfFrontPager() {
```

Comment:
I think it's better to just drop this function and [this call to setDisplayHomeAsUpEnabled](https://github.com/karyogamy/NewPipe/blob/0630423c8e41042521702a2b087391a1a1265bdc/app/src/main/java/org/schabi/newpipe/fragments/local/BaseLocalListFragment.java#L107..L113), and just let the already [implemented function in the activity](https://github.com/karyogamy/NewPipe/blob/0630423c8e41042521702a2b087391a1a1265bdc/app/src/main/java/org/schabi/newpipe/MainActivity.java#L293..L304) handle it.

This check should be used for other things such as: don't inflate menus or change the toolbar too much.